ably.d.ts

```diff
@@ -986,6 +986,16 @@ declare namespace Types {
     modes?: ChannelModes;
   }
 
+  /**
+   * Passes an attachment to a {@link ChannelBase} name to give a new derived channel
```

Comment:
I think this is always RealtimeChannelBase in this case?

What is meant by "passes an attachment"? I might be missing some context, but to me this should be something like "passes additional properties to a RealtimeChannelBase to produce a new derived channel"